src/components/articles/comments/ArticleComments.jsx

```diff
@@ -69,6 +74,41 @@ const ArticleComments = () => {
     setFailure(false);
   };
 
+  const updateVotes = (comment_id, inc_votes) => {
+    if (loginUser) {
+      setVotes(votes + inc_votes);
+      patchCommentVotes(comment_id, inc_votes)
+        .then((commentRes) => {
+          setVotes(0);
+          setSuccess(true);
+          if (inc_votes > 0) {
+            setMessage("You Upvoted the Comment! 👍");
+          } else {
+            setMessage("You Downvoted the Comment! 👎");
+          }
+          setArticleComments((currComments) => {
+            const newComments = currComments.filter(
+              (comment) => comment.comment_id !== comment_id
+            );
+            return [commentRes, ...newComments];
+          });
+        })
+        .catch((err) => {
+          setVotes(0);
+          setFailure(true);
+          setMessage("Something went wrong");
+        });
+      setSuccess(false);
+      setFailure(false);
+    } else {
+      setFailure(true);
+      setMessage("Please login to Vote");
+      setTimeout(() => {
+        setFailure(false);
+      }, 3000);
+    }
+  };
+
```

Comment:
Is this a copy paste of the function you used in the article vote page? If so i would consider extracting out this function or creating a component which handles voting as a whole which can be used in both cases.